src/cli.rs

```diff
@@ -84,18 +84,6 @@ pub struct InfoCliOptions {
     /// Show the email address of each author
     #[arg(long, short = 'E')]
     pub email: bool,
-    /// Count hidden files and directories
-    #[arg(long)]
-    pub include_hidden: bool,
-    /// Filters output by language type
-    #[arg(
-        long,
-        num_args = 1..,
-        default_values = &["programming", "markup"],
-        short = 'T',
-        value_enum,
-    )]
-    pub r#type: Vec<LanguageType>,
```

Comment:
Will this option be available eventually?

Reply:
That's one of the things I wanted to discuss. If we instruct users to use, for example,
```gitattributes
# .gitattributes
my-data-file.json gengo-detectable
```

Then they can have very fine control over what is and isn't reported. Instead of reporting all or no data files, prose files, etc., they can control which specific files are reported. Also it simplifies onefetch's code if onefetch doesn't have to handle this :wink:

Re-implementing the current behavior should be easily possible by using `gengo::Category`, though.

## Pros

- Simplify onefetch's code
- User has fine control over exactly which files to include/exclude

## Cons

- Asking a user to follow the configuration options of a separate tool can be obscure (I personally wasn't fond of asking users to use `.tokeignore` for this reason)